exercise3-1/src/camera_demo/src/main.cc

```diff
@@ -21,11 +21,8 @@ int main(int argc, char** argv) {
   RegisterThreadTracer(object_detector_tracer);
   RegisterThreadTracer(data_logger_tracer);
 
-  rclcpp::executors::SingleThreadedExecutor executor;
-
-  executor.add_node(camera_processing_node);
-  executor.add_node(actuation_node);
-  executor.spin();
+  // TODO: Create executor, add two nodes to it, and call spin() on executor
+  //       Find example code here: https://docs.ros.org/en/foxy/Concepts/About-Executors.html#basic-use
```

Comment:
Minor note: as we might not have reliable internet, this page may be difficult for some participant to reach.

That said, we can also show an example on the slides in big fonts, which should be good enough.